terraform/modules/galileo-eks/examples/network.tf

```diff
@@ -0,0 +1,221 @@
+resource "aws_vpc" "vpc" {
+  cidr_block = "192.168.0.0/16"
+
+  enable_dns_support   = true
+  enable_dns_hostnames = true
+
+  tags = {
+    Name = "galileo_${var.environment}_vpc"
+  }
+}
+
+# Create Public Subnets
+resource "aws_subnet" "public_subnet_a" {
+  vpc_id            = aws_vpc.vpc.id
+  cidr_block        = "192.168.0.0/19"
+  availability_zone = "us-west-2a"
+  map_public_ip_on_launch = true  # Enable auto-assign public IP
+
+  tags = {
+    Name = "galileo_${var.environment}_public_subnet_a"
+  }
+}
+
+resource "aws_subnet" "public_subnet_b" {
+  vpc_id            = aws_vpc.vpc.id
+  cidr_block        = "192.168.32.0/19"
+  availability_zone = "us-west-2b"
+  map_public_ip_on_launch = true
+
+  tags = {
+    Name = "galileo_${var.environment}_public_subnet_b"
+  }
+}
+
+resource "aws_subnet" "public_subnet_c" {
+  vpc_id            = aws_vpc.vpc.id
+  cidr_block        = "192.168.64.0/19"
+  availability_zone = "us-west-2c"
+  map_public_ip_on_launch = true
+
+  tags = {
+    Name = "galileo_${var.environment}_public_subnet_c"
+  }
+}
```

Comment:
We need 3 public subnets 👀?

Reply:
I think only 2, but I was mimicking the networking setup done by `eksctl` as close as possible